docs/debuginfod.md

```diff
@@ -0,0 +1,37 @@
+# debuginfod client
+
+In order to symbolize ingested profiles, Parca needs to have debug information
+for the binaries _(which binaries?)_ that are being profiled. Debug information
+can typically comprises ELF object files, DWARF debug data, and source code
+required for symbolization. However, sometimes, production codebases use
+compiler optimisations which strip the debug information from the binaries.
```

Comment:
Stipping debug information is not typically a performance optimization, it's usually just done to minimize the size of the binary. I would also note that this is most prominently the case in packages distributed by the various Linux distros, as that's where debuginfod is most useful.